source/classes.tex

```diff
@@ -4201,6 +4179,27 @@
 \tcode{circle::draw()} must be provided.
 \end{example}
 
+\pnum
+\begin{note}
+In general, an abstract class type can only be used
+\begin{itemize}
+\item
+as the referred to type in a reference,
```

Comment:
No. That's awkward and needs at least a hyphen, but even better, needs a rephrasing. Maybe "referenced type" and "pointee type" or so?